### PR TITLE
Cleanup and fix chefspec tests so they run and pass

### PR DESCRIPTION
**Pull request by ramereth**
**NOTE: This doesn't include complete coverage, but the tests now at least run and pass**

### CODE REVIEW

File: spec/default_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require_relative 'spec_helper'
+
+describe 'osl-app::default' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_7).converge('sudo', described_recipe)
+  end
+  include_context 'common_stubs'
+  it do
```
Comment (Kennric) [inline spec/default_spec.rb]: 'it' probably should have an argument, "runs chef without failure" or similar

Comment (ramereth) [inline spec/default_spec.rb]: Our new standard is to let chefspec do this automatically unless we want to have an explicit message. As it stands now, here's what it prints:

```
 osl-app::default
  should not raise Exception
```

Comment (Kennric) [inline spec/default_spec.rb]: ok, that's fine with me then

**Comment by Kennric** [comment]
+1

**Comment by ramereth** [comment]
!bump patch minor

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.4.6.  Have a nice day!

**Pull request merged**